opendrift/models/oceandrift.py

```diff
@@ -435,7 +435,7 @@ def vertical_mixing(self, store_depths=False):
         else:
             return None
 
-    def animate_vertical_distribution(self, depths=None, maxdepth=None, bins=50, filename=None):
+    def animate_vertical_distribution(self, depths=None, maxdepth=None, bins=50, filename=None, subsamplingstep=1):
```

Comment:
Looks very useful, it would be very good with some documentation in this docstring about what the parameter does. See e.g. `def  plot` for a messy example :D